BE/src/main/java/louie/dong/airbnb/accommodation/AccommodationService.java

```diff
@@ -26,91 +27,63 @@
 @Service
 @RequiredArgsConstructor
 @Transactional(readOnly = true)
+@Slf4j
 public class AccommodationService {
 
-	private final AccommodationRepository accommodationRepository;
-	private final WishRepository wishRepository;
-
-	public AccommodationPriceResponse findPrices(String country) {
-		List<Integer> prices = accommodationRepository.findByAccommodationPrices(country);
-		return new AccommodationPriceResponse(calculateAverage(prices), prices);
-	}
-
-	public AccommodationDetailResponse findById(Long id) {
-		Accommodation accommodation = getAccommodation(id);
-		boolean wish = wishRepository.existsByAccommodationId(id);
-		return new AccommodationDetailResponse(accommodation, wish);
-	}
-
-	public AccommodationDetailPriceResponse findDetailPrice(Long id,
-		AccommodationDetailPriceRequest accommodationDetailPriceRequest) {
-		Accommodation accommodation = getAccommodation(id);
-		LocalDate checkIn = accommodationDetailPriceRequest.getCheckIn();
-		LocalDate checkOut = accommodationDetailPriceRequest.getCheckOut();
-
-		int date = (int) checkIn.until(checkOut, ChronoUnit.DAYS);
-		int totalPrice = accommodation.getPrice() * date;
-		double discountRate = getDiscountRate(date) * 0.01;
-		int discountPrice = (int) (totalPrice * discountRate);
-		int finalPrice = totalPrice - discountPrice + accommodation.getCleaningFee()
-			+ accommodation.getServiceFee() + accommodation.getAccommodationFee();
-
-		return new AccommodationDetailPriceResponse(accommodation.getPrice(), date, totalPrice,
-			WEEKLY.getDiscountRate(), discountPrice, accommodation.getCleaningFee(),
-			accommodation.getServiceFee(), accommodation.getAccommodationFee(), finalPrice);
-	}
-
-	public AccommodationSearchResponse findAccommodations(
-		AccommodationSearchRequest accommodationSearchRequest) {
+    private final AccommodationRepository accommodationRepository;
+    private final WishRepository wishRepository;
+
+    public AccommodationPriceResponse findPrices(String country) {
+        List<Integer> prices = accommodationRepository.findPricesByAccommodation(country);
+        return new AccommodationPriceResponse(calculateAverage(prices), prices);
+    }
```

Comment:
컬렉션을 가공해야 하는 경우 일급컬렉션을 사용해주면 서비스 레이어의 로직을 최소화 할 수 있습니다.

```suggestion
    public AccommodationPriceResponse findPrices(String country) {
        Prices prices = new Prices(accommodationRepository.findPricesByAccommodation(country));
        return new AccommodationPriceResponse(prices.average(), prices);
    }
```

단순 평균값 계산에 적용하기엔 과해 보이지만,
다른 로직(ex. 짝수 id 만  계산)이 필요하다면 일급 컬렉션을 사용해 보는걸 추천합니다 :)

[일급 콜렉션](https://tecoble.techcourse.co.kr/post/2020-05-08-First-Class-Collection/)